### PR TITLE
hub: preparation for mock config autodetection

### PR DESCRIPTION
**Pull request by lzaoral**

### CODE REVIEW

File: osh/worker/tasks/task_errata_diff_build.py [hunk 1]
```diff
@@ -56,14 +56,13 @@ def run(self):
 
         scanning_args = self.hub.worker.get_scanning_args(profile)
         add_args = scanning_args.get('csmock_args', '')
-        koji_profile = scanning_args.get('koji_profile', 'koji')
 
         with CsmockRunner() as runner:
             results, retcode = runner.koji_analyze(scanning_args['analyzers'],
-                                                   build,
+                                                   build['nvr'],
```
Comment (kdudka) [inline osh/worker/tasks/task_errata_diff_build.py:62]: This seems to trigger a traceback if an existing worker connects to the updated hub:
```py
Traceback (most recent call last):
  File "/usr/lib/python3.9/site-packages/kobo/worker/taskmanager.py", line 423, in run_task
    task.run()
  File "/usr/lib/python3.9/site-packages/osh/worker/tasks/task_errata_diff_build.py", line 62, in run
    results, retcode = runner.koji_analyze(scanning_args['analyzers'],
  File "/usr/lib/python3.9/site-packages/osh/worker/csmock_runner.py", line 168, in koji_analyze
    run(download_cmd, stdout=True, return_stdout=False, buffer_size=2,
  File "/usr/lib64/python3.9/subprocess.py", line 368, in check_call
    retcode = call(*popenargs, **kwargs)
  File "/usr/lib64/python3.9/subprocess.py", line 349, in call
    with Popen(*popenargs, **kwargs) as p:
  File "/usr/lib64/python3.9/subprocess.py", line 951, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/usr/lib64/python3.9/subprocess.py", line 1770, in _execute_child
    self.pid = _posixsubprocess.fork_exec(
TypeError: expected str, bytes or os.PathLike object, not dict
Frame _execute_child in /usr/lib64/python3.9/subprocess.py at line 1770
<CODE>
    1763                     else:
    1764                         # This matches the behavior of os._execvpe().
    1765                         executable_list = tuple(
    1766                             os.path.join(os.fsencode(dir), executable)
    1767                             for dir in os.get_exec_path(env))
    1768                     fds_to_keep = set(pass_fds)
    1769                     fds_to_keep.add(errpipe_write)
--> 1770                     self.pid = _posixsubprocess.fork_exec(
    1771                             args, executable_list,
    1772                             close_fds, tuple(sorted(map(int, fds_to_keep))),
    1773                             cwd, env_list,
    1774                             p2cread, p2cwrite, c2pread, c2pwrite,
    1775                             errread, errwrite,
    1776                             errpipe_read, errpipe_write,
</CODE>
```

**Review by kdudka — COMMENTED** [review]
There seems to be an extra space in the commit message of 5af1b963c17978af200819857bb4fc749fd55a3a: `passing of  nvr`.

File: osh/hub/settings.py [hunk 2]
```diff
@@ -179,7 +179,7 @@
 LOGIN_REDIRECT_URL = 'index'
 LOGOUT_REDIRECT_URL = 'index'
 
-VALID_TASK_LOG_EXTENSIONS = ['.log', '.ini', '.err', '.out', '.js', '.txt']
+VALID_TASK_LOG_EXTENSIONS = ['.log', '.ini', '.err', '.out', '.js', '.txt', '.cfg']
```
[the remark this answers is not in the document]
Comment (lzaoral) [reply osh/hub/settings.py:182]: You're right, I'll sort everything in `settings*.py` in a separate PR.

**Comment by lzaoral** [comment]
Closing in favour of #228.

**Pull request closed without merging**